pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1210,6 +1210,66 @@ func (configgen *ConfigGeneratorImpl) generateManagementListeners(node *model.Pr
 	return listeners
 }
 
+// buildSidecarVirtualOutboundListener creates the outbound virtual listener
+// which receives traffic for IPTABLES redirect
+func (configgen *ConfigGeneratorImpl) buildSidecarVirtualOutboundListener(env *model.Environment,
+	node *model.Proxy,
+	push *model.PushContext, proxyInstances []*model.ServiceInstance,
+	ipTablesListener *xdsapi.Listener) *xdsapi.Listener {
+
+	hostname := model.Hostname(util.BlackHoleCluster)
+	mesh := env.Mesh
+	redirectPort := &model.Port{
+		Port:     int(mesh.ProxyListenPort),
+		Protocol: model.ProtocolTCP,
+	}
+
+	if len(ipTablesListener.FilterChains) < 1 || len(ipTablesListener.FilterChains[0].Filters) < 1 {
+		return ipTablesListener
+	}
+
+	filter := ipTablesListener.FilterChains[0].Filters[0]
+	if isAllowAnyOutbound(node) {
+		hostname = util.PassthroughCluster
+	}
+
+	pluginParams := &plugin.InputParams{
+		ListenerProtocol:           plugin.ListenerProtocolTCP,
+		DeprecatedListenerCategory: networking.EnvoyFilter_DeprecatedListenerMatch_SIDECAR_OUTBOUND,
```

Comment:
use the newer ListenerCategory

Reply:
I don't see any example usage of the new category. Everywhere `DeprecatedListenerCategory` is used instead of `ListenerCategory` in listener.go. Can you point me to the new usage?